<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Deivces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Points</string>
    </property>
   </widget>
   <widget class="QLabel" name="iplbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxlbl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Device Instance</string>
    </property>
   </widget>
   <widget class="QLabel" name="minlbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Device Instance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pointSelect">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BO</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="pointNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="pointlbl">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Point:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="binaryInput">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Active</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NULL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inactive</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="analogInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="valuelbl">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="submasklbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Subnet Mask:</string>
    </property>
   </widget>
   <widget class="QLabel" name="portlbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="globallbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Global Controller IP (Optional):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="prioritylbl">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Priority:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ipInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subMaskInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/24</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>47808</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="globalIpInput">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0.0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4194303</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
